src/Akka.Persistence.Redis.Tests/Query/RedisPersistenceIdsSpec.cs

```diff
@@ -39,6 +45,49 @@ public RedisPersistenceIdsSpec(ITestOutputHelper output, RedisFixture fixture) :
         {
             ReadJournal = Sys.ReadJournalFor<RedisReadJournal>(RedisReadJournal.Identifier);
         }
+        [Fact]
+        public override Task ReadJournal_should_deallocate_AllPersistenceIds_publisher_when_the_last_subscriber_left()
+        {
+            if (AllocatesAllPersistenceIDsPublisher)
+            {
+                var journal = ReadJournal.AsInstanceOf<IPersistenceIdsQuery>();
+
+                Setup("a", 1);
+                Setup("b", 1);
+
+                var source = journal.PersistenceIds();
+                var probe =
+                    source.RunWith(this.SinkProbe<string>(), Materializer);
+                var probe2 =
+                    source.RunWith(this.SinkProbe<string>(), Materializer);
+
+                var fieldInfo = journal.GetType()
+                    .GetField("_persistenceIdsPublisher",
+                        BindingFlags.NonPublic | BindingFlags.Instance);
+                Assert.True(fieldInfo != null);
+
+                // Assert that publisher is running.
+                probe.Within(TimeSpan.FromSeconds(10), () => probe.Request(10)
+                    .ExpectNextUnordered("a", "b")
+                    .ExpectNoMsg(TimeSpan.FromMilliseconds(200)));
+
+                probe.Cancel();
+
+                // Assert that publisher is still alive when it still have a subscriber
+                //Assert.True(fieldInfo.GetValue(journal) is IPublisher<string>);
```

Comment:
This makes this test basically useless

Reply:
`private IPublisher<string> _persistenceIdsPublisher;` is needed for that test to pass.
This makes that unnecessary:
```csharp
public Source<string, NotUsed> PersistenceIds() =>
            Source.FromGraph(new PersistenceIdsSource(_redis, _database, _system));
```
the SQL implementation had that

Reply:
You left all the code in there, future coders will be confused when they read your code.
A better way is to clear all the codes and add the `Skip` parameter to the `Fact` attribute with a proper explanation on why it is being skipped.

Reply:
What about clearing it and not using the `Skip` seeing it is a test that needs to run?